src/qt/glcanvas.cpp

```diff
@@ -378,6 +378,15 @@ wxGLCanvas::wxGLCanvas(wxWindow *parent,
     Create(parent, id, pos, size, style, name, attribList, palette);
 }
 
+wxGLCanvas::~wxGLCanvas()
+{
+        // Avoid sending further signals (i.e. if deleting the current page)
+        m_qtWindow->blockSignals(true);
+        // Reset the pointer to avoid handling pending event and signals
+        //QtStoreWindowPointer( GetHandle(), NULL );
+
```

Comment:
Please don't leave commented out code, it should either be uncommented or removed.

Reply:
Agreed. Will fix.